test/acceptance/create_webdriver.robot

```diff
@@ -29,6 +29,40 @@ Create Webdriver With Bad Keyword Argument Dictionary
     Run Keyword And Expect Error    kwargs must be a dictionary.
     ...    Create Webdriver    Firefox    kwargs={'spam': 'eggs'}
 
+Get Browser Capabilities Using Dictionary
+    [Documentation]    Gets current browser capabilities from WebDriver
+    [Setup]    Set Driver Variables
+    Create Webdriver    ${DRIVER_NAME}
+    Go To    ${FRONT PAGE}    # If we don't do this then no version details
```

Comment:
Could this be fixed so that `Go To` would be needed? If not, then this limitation should probably be documented.